cdn/tests/unit.tftest.hcl

```diff
@@ -106,3 +106,36 @@ run "domain_length_validation_tests_succeed_with_empty_config" {
   }
 }
 
+run "cdn_read_timeout_is_set_to_30_seconds_by_default" {
+  command = plan
+
+  assert {
+    condition = [
+      for k in aws_cloudfront_distribution.standard["dev.my-application.uktrade.digital"].origin :
+      k.custom_origin_config[0].origin_read_timeout
+      if k.domain_name == "internal.env.app.uktrade.digital"
+    ][0] == 30
+    error_message = "Should be: 30 seconds"
+  }
+}
+
+run "cdn_read_timeout_is_set_to_config_specified_value_when_provided" {
+  command = plan
+
+  variables {
+    config = {
+      cdn_domains_list    = { "dev.my-application.uktrade.digital" : ["internal", "my-application.uktrade.digital"] }
+      cdn_timeout_seconds = 60
+    }
+  }
+
+
+  assert {
+    condition = [
```

Comment:
@james-francis-MT and I would be curious to know if there's a more elegant way of getting the `origin_read_timeout` value here. We ran into issues in accessing this value owing to it being a set type, and this seemed to be the only way we could extract the timeout.

Reply:
I think we're stuck with it. I had to do similar [here](https://github.com/uktrade/terraform-platform-modules/pull/254/files#diff-be639a729ee265634f6b35550c54684eb315c7e7c1306efb6ca0fe1695664e28R427-R435) and it's a disgusting syntax.

Reply:
You might be able to do something like this, which I am doing for testing the contents of `aws_iam_policy_document`s...

```
  assert {
    condition     = strcontains(jsonencode(data.aws_iam_policy_document.s3_migration_policy_document.statement[1].resources), "test-destination-bucket-arn")
    error_message = "Should contain: test-destination-bucket-arn"
  }
```

Still not brilliant, but perhaps a bit more readable.

Reply:
Didn't have much luck in getting this to work: continued coming across the set type error. I'm happy to keep the syntax as is for now, tbh